src/main/java/com/dft/veracross/VeracrossSDK.java

```diff
@@ -0,0 +1,103 @@
+package com.dft.veracross;
+
+import com.dft.veracross.credentials.VeracrossCredentials;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.SneakyThrows;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+@AllArgsConstructor
+@Builder(builderMethodName = "newBuilder", toBuilder = true)
+public class VeracrossSDK {
+
+    private final VeracrossCredentials credentials;
+    int MAX_ATTEMPTS = 50;
+    int TIME_OUT_DURATION = 60000;
+    private final HttpClient client;
+    private static final String AUTHORIZATION = "Authorization";
+    private static final String BASE_ENDPOINT = "https://api.veracross.com/api-sandbox/v3";
+
+    public VeracrossSDK(VeracrossCredentials credentials) {
+        this.credentials = credentials;
+        client = HttpClient.newBuilder()
+                .followRedirects(HttpClient.Redirect.ALWAYS)
+                .version(HttpClient.Version.HTTP_1_1)
+                .connectTimeout(Duration.ofSeconds(20))
+                .build();
+    }
+
+    @SneakyThrows
```

Comment:
Remove @SneakyThrows after changing new URI to URI.create 
check next comment.

Reply:
Do same for other methods.

Reply:
Changed to URI.create and removed @SneakyThrows.